01_login.php

```diff
@@ -42,7 +42,6 @@
 		<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Lato">
 		<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
 		<link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
-		<link href='https://fonts.googleapis.com/css?family=Alegreya' rel='stylesheet'>
```

Comment:
deleted font code: since my stakeholders and I prefer the font in version3 (helvetica font for headings)